src/components/shared/SearchInput.js

```diff
@@ -60,9 +60,9 @@ class SearchInput extends Component<Props> {
     if (this.$input) {
       const input = this.$input;
       input.focus();
-      if (input.value != "") {
-        input.select();
-      }
+
+      const selectStart = /^[@:#]/.test(input.value) ? 1 : 0;
+      input.setSelectionRange(selectStart, input.value.length + 1);
```

Comment:
SearchInput is a shared component now so i dont think it should know about the prefixes... `/^[@:#]` 

maybe we should have something like `showErrorEmoji` for disabling the select behavior...

Maybe we need to begin asking weather Search Input should be a shared component or just a part of quick open.